File: src/main/java/org/jukeboxmc/potion/HungerEffect.java
package org.jukeboxmc.potion;

import org.jetbrains.annotations.NotNull;
import org.jukeboxmc.entity.EntityLiving;

import java.awt.*;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class HungerEffect extends Effect {

    @Override
    public int getId() {
        return 17;
    }

    @Override
    public @NotNull EffectType getEffectType() {
        return EffectType.HUNGER;
    }

    @Override
    public @NotNull Color getEffectColor() {
        return new Color( 46, 139, 87 );
    }

    @Override
    public void apply( EntityLiving entityLiving ) {

    }

    @Override
    public void update( long currentTick ) {

    }

    @Override
    public void remove( EntityLiving entityLiving ) {

    }
}

File: JukeboxMC-Server/src/main/kotlin/org/jukeboxmc/server/effect/HungerEffect.kt
package org.jukeboxmc.server.effect

import org.jukeboxmc.api.effect.EffectType
import org.jukeboxmc.server.entity.JukeboxEntityLiving
import java.awt.Color

class HungerEffect : JukeboxEffect() {

    override fun getId(): Int {
        return 17
    }

    override fun getEffectType(): EffectType {
        return EffectType.HUNGER
    }

    override fun getEffectColor(): Color {
        return Color(46, 139, 87)
    }

    override fun apply(entityLiving: JukeboxEntityLiving) {

    }

    override fun update(currentTick: Long) {

    }

    override fun remove(entityLiving: JukeboxEntityLiving) {

    }
}